cmd/user-agent/src/pages/chapi/Get.vue

```diff
@@ -13,6 +13,7 @@ SPDX-License-Identifier: Apache-2.0
 
     import DIDAuthForm from "./DIDAuth.vue";
     import GetCredentialsForm from "./GetCredentials.vue";
+    import GetVP from "./GetVP.vue";
```

Comment:
for follow up, may be better name --> like 'PresentationDefQuery.vue',  'GetCredentialsByQuery.vue'. Just to avoid confusion since GetVP name is too generic since CHAPI always gets VP.

Reply:
done change to PresentationDefQuery